evolver/settings.py

```diff
@@ -34,11 +34,27 @@ class Settings(BaseSettings):
 
 
 class AppSettings(BaseSettings):
-    CONFIG_FILE: Path = Path("evolver.yml")  # in current directory
+    def find_project_root(self, marker: str = "pyproject.toml") -> Path:
```

Comment:
So this already exists [evolver.util.find_package_location](https://github.com/ssec-jhu/evolver-ng/blob/main/evolver/util.py#L10).

Reply:
ah nice.